Changelog.md

```diff
@@ -11,6 +11,7 @@ Breaking Changes:
  * Type Checker: Disallow arithmetic operations for Boolean variables.
  * Disallow trailing dots that are not followed by a number.
  * Remove assembly instructions ``sha3`` and ``suicide``
+ * Remove obsolete ``std` directory from the Solidity repository. This means accessing ``https://github.com/etheruem/soldity/std/*.sol`` will not be possible.
```

Comment:
There're some typos and missing back-ticks which breaks the markdown rendering. I'd go for (but fine with how it is): 

Remove obsolete ``std`` directory from the Solidity repository. Accessing ``https://github.com/ethereum/solidity/std/*.sol`` is not be possible any more.